codec-http/src/main/java/io/netty/handler/codec/http/DefaultHttpHeaders.java

```diff
@@ -42,7 +42,7 @@
 /**
  * Default implementation of {@link HttpHeaders}.
  */
-public class DefaultHttpHeaders extends HttpHeaders {
+public class DefaultHttpHeaders extends HttpHeaders implements Cloneable {
```

Comment:
this class doesn't implement `clone` so should we get rid of the `implements Cloneable`?

Reply:
@Scottmitch doh.. good catch, thats a left-over as I first wanted to just use `clone()` for it.